src/components/ActivityTimeline/index.js

```diff
@@ -26,10 +105,32 @@ ActivityTimeline.propTypes = {
     className: PropTypes.string,
     /** An object with custom style applied to the outer element. */
     style: PropTypes.object,
+    /** If true, expands multiples TimelineMarkers.
+     * This value defaults to false. */
+    multiple: PropTypes.bool,
+    /** The variant changes the appearance of the timeline. Accepted variants include
+     * default and accordion. */
+    variant: PropTypes.oneOf(['default', 'accordion']),
+    /** It contain the name of the TimelineMarker that is expanded.
+     * It is an array of string when multiple is true,
+     * or a string when when multiple is false.
+     * It must match the name of the TimelineMarker. */
+    activeSectionNames: PropTypes.oneOfType([
+        PropTypes.arrayOf(PropTypes.string),
+        PropTypes.string,
+    ]),
+    /** Action fired when a TimelineMarker is selected.
+     * The event params include the `name` of the selected TimelineMarker. */
```

Comment:
we need to explain that this props (multiple, activeSectionNames and onToggleSection) are only used for accordion variant